kit/servo_board.md

```diff
@@ -16,14 +16,15 @@ Board Diagram
 -------------
 <img src="{{ site.baseurl }}/images/content/kit/servo_board_v4_diagram.png" alt="A diagram of a servo board" />
 
-<!--
 Indicators
 ----------
 
-|   LED           | Meaning                | Initial power-up state
-|-----------------|------------------------|----------------------
-| Power           | The board is powered   | On
--->
+| LED                   | Meaning                                                                                          | Initial power-up state |
+| --------------------- | ------------------------------------------------------------------------------------------------ | ---------------------- |
+| Power <!-- DS1 -->    | The board is powered over USB.                                                                   | On                     |
+| 5.5V On <!-- DS15 --> | There is 5.5V power on the board. This usually indicates that the 12V connector rail is powered. | Off                    |
+| Aux On <!-- DS16 -->  | There is auxiliary power on the board.                                                           | Off                    |
+| STATUS\|ERROR <!-- DS10 --> | Blue when the board has successfully booted<br />Flickering red when changing servo positions<br />Solid pink if an error has occurred (often 12V power being lost) | Blue |
```

Comment:
What is this flickering refering to? Looking at the frmware this is only during the interrupt routine which under normal operation should not be on long enough to be visible.

Reply:
Ah, ok. I'd assumed it was going to be visible.

Reply:
76ba883c590a3d0779899124d57c81efbefde2d5